controller-server/src/main/java/com/yahoo/vespa/hosted/controller/Application.java

```diff
@@ -199,6 +199,15 @@ public ApplicationActivity activity() {
     /** Returns the set of deploy keys for this application. */
     public Set<PublicKey> deployKeys() { return deployKeys; }
 
+    private static Optional<ApplicationVersion> requireNotUnknown(Optional<ApplicationVersion> latestVersion) {
```

Comment:
requireKnown? 🙂

Reply:
Was wondering about that :p

Reply:
In this case I think the current name is more precise, since `unknown` is a specific instance.